src/main/java/AccountManager.java

```diff
@@ -0,0 +1,51 @@
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class AccountManager {
+
+    private HashMap<String, Person> users;
+
+    public AccountManager() {
+        users = new HashMap<>();
+    }
+
+    public boolean createAccount() {
+
+        AccountCreator creator = new AccountCreator();
+        Person user = new Person();
+
+        Scanner in = new Scanner (System.in);
```

Comment:
this looks like it would be interacting directly with the user, and i don't think AccountManager is in the appropriate layer of clean architecture to be allowed to do that -- could pass in the required values as parameters instead?

Reply:
I have changed the required values to parameters.